types/index.d.ts

```diff
@@ -166,15 +164,10 @@ declare namespace fastifySwaggerUi {
     configUrl?: string | undefined;
 
     /**
-     * REQUIRED if domNode is not provided. The ID of a DOM element inside which SwaggerUI will put its user interface.
+     * The ID of a DOM element inside which SwaggerUI will put its user interface.
```

Comment:
remove the ? to make it required.

dom_id: string | undefined;

Reply:
It shouldn't be required to override `dom_id`. It's already provided by `lib/swagger_initializer.js`

Reply:
ah, ok